src/library-authoring/component-info/ComponentInfo.tsx

```diff
@@ -184,7 +184,9 @@ const ComponentInfo = () => {
             >
               {intl.formatMessage(messages.publishComponentButtonTitle)}
             </Button>
-            <ComponentMenu usageKey={usageKey} />
+            <ComponentMenu
+              usageKey={usageKey}
+            />
```

Comment:
NIT: we can remove this change.

Reply:
Updated: https://github.com/openedx/frontend-app-authoring/pull/1906/commits/28e5cf4fa9fb7f9f9a6c7538c276b68e5c8780f2